docker-compose.yml

```diff
@@ -125,7 +108,8 @@ services:
       DATABASE_NAME: ${DB_NAME?}
       DATABASE_USER: ${DB_USER?}
       DATABASE_PASSWORD: ${DB_PASS?}
-      COCKROACH_HOST: roach-0:${DB_PORT?}
+      COCKROACH_HOST: roach-0
```

Comment:
Maybe we can use `${DB_HOST?}` instead of hardcoded `roach-0` and set that once in the env files

Reply:
yep, fixed in d3613d3